sim/side.ts

```diff
@@ -629,17 +629,24 @@ export class Side {
 
 			switch (choiceType) {
 			case 'move':
-				let targetLoc = 0;
-				if (/\s-?[1-3]$/.test(data)) {
-					targetLoc = parseInt(data.slice(-2), 10);
-					data = data.slice(0, data.lastIndexOf(' '));
+				let [targetLoc, willMega, willUltra, willZ] = [0, '', '', ''];
+				while (true) {
+					if (/\s-?[1-3]$/.test(data)) {
+						targetLoc = parseInt(data.slice(-2), 10);
+						data = data.slice(0, data.lastIndexOf(' '));
```

Comment:
```suggestion
						data = data.slice(0, -2).trim();
```

In hindsight, this is probably easier to read.